config/resourcefulbees/resources/assets/resourcefulbees/lang/zh_cn.json

```
@@ -188,25 +188,25 @@
   "item.resourcefulbees.wither_bee_spawn_egg": "凋灵蜜蜂刷怪蛋",
   "entity.resourcefulbees.wither_bee": "凋灵蜜蜂",
 
-  "block.resourcefulbees.withered_honeycomb_block": "枯萎蜜脾块",
-  "item.resourcefulbees.withered_honeycomb": "枯萎蜜脾",
-  "item.resourcefulbees.withered_bee_spawn_egg": "枯萎蜜蜂刷怪蛋",
-  "entity.resourcefulbees.withered_bee": "枯萎蜜蜂",
+  "block.resourcefulbees.withered_honeycomb_block": "凋零蜜脾块",
+  "item.resourcefulbees.withered_honeycomb": "凋零蜜脾",
+  "item.resourcefulbees.withered_bee_spawn_egg": "凋零蜜蜂刷怪蛋",
+  "entity.resourcefulbees.withered_bee": "凋零蜜蜂",
 
   "block.resourcefulbees.beeper_honeycomb_block": "蜂力怕蜜脾块",
   "item.resourcefulbees.beeper_honeycomb": "蜂力怕蜜脾",
-  "item.resourcefulbees.beeper_bee_spawn_egg": "蜂力怕蜜蜂刷怪蛋",
-  "entity.resourcefulbees.beeper_bee": "蜂力怕蜜蜂",
+  "item.resourcefulbees.beeper_bee_spawn_egg": "蜂力怕刷怪蛋",
+  "entity.resourcefulbees.beeper_bee": "蜂力怕",
 
   "block.resourcefulbees.ender_honeycomb_block": "末影蜜脾块",
   "item.resourcefulbees.ender_honeycomb": "末影蜜脾",
   "item.resourcefulbees.ender_bee_spawn_egg": "末影蜜蜂刷怪蛋",
   "entity.resourcefulbees.ender_bee": "末影蜜蜂",
 
-  "block.resourcefulbees.guardian_honeycomb_block": "守护蜜脾块",
-  "item.resourcefulbees.guardian_honeycomb": "守护蜜脾",
-  "item.resourcefulbees.guardian_bee_spawn_egg": "守护蜜蜂刷怪蛋",
-  "entity.resourcefulbees.guardian_bee": "守护蜜蜂",
+  "block.resourcefulbees.guardian_honeycomb_block": "守护者蜜脾块",
+  "item.resourcefulbees.guardian_honeycomb": "守护者蜜脾",
+  "item.resourcefulbees.guardian_bee_spawn_egg": "守护者蜜蜂刷怪蛋",
+  "entity.resourcefulbees.guardian_bee": "守护者蜜蜂",
```

[守卫者](https://minecraft.fandom.com/zh/wiki/%E5%AE%88%E5%8D%AB%E8%80%85?variant=zh)
```suggestion
  "block.resourcefulbees.guardian_honeycomb_block": "守卫者蜜脾块",
  "item.resourcefulbees.guardian_honeycomb": "守卫者蜜脾",
  "item.resourcefulbees.guardian_bee_spawn_egg": "守卫者蜜蜂刷怪蛋",
  "entity.resourcefulbees.guardian_bee": "守卫者蜜蜂",
```